pom.xml

```diff
@@ -130,8 +130,19 @@
 		<!-- Test configuration -->
 		<skipITs>true</skipITs>
 
-		<ice.version>3.6.4</ice.version>
+		<ice.version>3.6.5</ice.version>
 		<glacier2.version>${ice.version}</glacier2.version>
+
+		<omero-blitz.version>5.5.10</omero-blitz.version>
+		<omero-common.version>5.5.9</omero-common.version>
+		<omero-dsl-plugin.version>5.5.2</omero-dsl-plugin.version>
+		<omero-gateway.version>5.6.9</omero-gateway.version>
+		<omero-model.version>5.6.5</omero-model.version>
+		<omero-renderer.version>5.5.9</omero-renderer.version>
+		<omero-romio.version>5.6.4</omero-romio.version>
+		<omero-server.version>5.6.3</omero-server.version>
```

Comment:
These all need to make their way into a pom-scijava release (I'd be happy to do an immediate 32.0.0-beta-5 as needed), so that this component can be free and clear of any version property overrides. Otherwise, it creates problems with downstream projects depending on imagej-omero in their POMs.